components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/util/APIMgtConstants.java

```diff
@@ -374,7 +374,7 @@
         public static final String API_CREATE = "API_CREATE";
         public static final String API_UPDATE = "API_UPDATE";
         public static final String API_DELETE = "API_DELETE";
-        public static final String API_PUBLISH = "API_PUBLISH";
+        public static final String API_STATE_CHANGE = "API_STATE_CHANGE";
```

Comment:
This is for API Manager 3.0 which is under development with C5. We haven't done a major release and this change comes under API Manager 3.0.0-M3. The development is still on going.